.azure/azure-pipelines-daily.yml

```diff
@@ -85,6 +85,19 @@ jobs:
     fetchDepth: 25
   - bash: ./checker/bin-devel/test-cftests-junit.sh
     displayName: test-cftests-junit.sh
+- job: junit_jdk25
+  dependsOn:
+   - canary_jobs
+   - junit_jdk24
+  pool:
+    vmImage: 'ubuntu-latest'
+  container: mdernst/cf-ubuntu-jdk25:latest
+  timeoutInMinutes: 70
+  steps:
+  - checkout: self
+    fetchDepth: 25
+  - bash: ./checker/bin-devel/test-cftests-junit.sh
```

Comment:
Can you change this line to 
 export ORG_GRADLE_PROJECT_jdkTestVersion=25  && ./checker/bin-devel/test-cftests-junit.sh, where the Java version is set correctly based on the test?

Alternatively, we could read the JAVA_HOME version and add the export to each test script.